packages/design-picker/src/utils/designs.ts

```diff
@@ -33,7 +33,7 @@ export const getDesignPreviewUrl = (
 			? options.viewport_height || DEFAULT_VIEWPORT_HEIGHT
 			: undefined,
 		source_site: 'patternboilerplates.wordpress.com',
-		use_screenshot_overrides: options.use_screenshot_overrides,
+		use_screenshot_overrides: options.use_screenshot_overrides || true,
```

Comment:
This feels like it should either be `?? true` or just `true`, as the `||` will fall through to `true` if the incoming value is falsey.

Reply:
Good catch.

What do you think about changing it to:

```
...( options.use_screenshot_overrides && {
			use_screenshot_overrides: options.use_screenshot_overrides,
		} ),
```

This way its added only when necessary and I can update the design-carousel item component:

```
<MShotsImage
	url={ getDesignPreviewUrl( design, {
	     use_screenshot_overrides: true,
	} ) }
	options={ options }
	alt={ design.title }
        aria-labelledby=""
/>
```

Reply:
I think that should work!